### ISSUE TITLE
📜 Multiple instances of image-builder fail to send logs to cloudwatch

### ISSUE THREAD

**Issue by major** [osbuild/image-builder#96]
We are seeing this error in OpenShift:

```
Failed to fire hook: InvalidSequenceTokenException: The given sequenceToken is invalid. The next expected sequenceToken is: 49612946959762469195597770664378024234801679062591571890
{
  RespMetadata: {
    StatusCode: 400,
    RequestID: "da491f7a-ae72-4317-b52b-5038030bfd19"
  },
  ExpectedSequenceToken: "49612946959762469195597770664378024234801679062591571890",
  Message_: "The given sequenceToken is invalid. The next expected sequenceToken is: 49612946959762469195597770664378024234801679062591571890"
}
```

The issue seems to be related to one of the pods sending a message and incrementing its log sequence token, but then the other pod sends a log with the old sequence token. More details are in this [stack overflow post](https://stackoverflow.com/questions/34525955/aws-cloudwatch-invalidsequencetokenexception).

**Comment by major** [comment]
Asking in `#sd-app-sre` in [Slack](https://coreos.slack.com/archives/CCRND57FW/p1610459383049700) to find out what a solution should be.

**Comment by Gundersanne** [comment]
Fixed in #97, using the hostname (on openshift that results in the pod name, so image-builder-qwd9212) as stream. Having separate streams where each has it's own sequencetoken should fix the problem.
